pingora-core/src/protocols/tls/rustls/stream.rs

```diff
@@ -216,9 +216,15 @@ impl<T> UniqueID for TlsStream<T>
 where
     T: UniqueID,
 {
+    #[cfg(not(target_os = "windows"))]
```

Comment:
```suggestion
    #[cfg(unix)]
```

To keep it consistent with the other cfg statements